### PR TITLE
Model Builder | Updates to environment and compute

### PR DESCRIPTION
**Pull request by luisquintanilla**
Add information on compute priority

Preview Link (Compute Priority) - https://review.docs.microsoft.com/en-us/dotnet/machine-learning/resources/azure-training-concepts-model-builder?branch=pr-en-us-18731#what-is-an-azure-machine-learning-compute

[AB#1725142](https://dev.azure.com/mseng/677da0fb-b067-4f77-b89b-f32c12bb8617/_workitems/edit/1725142)

### CODE REVIEW

**Comment by luisquintanilla** [comment]
@beccamc can you take a look what you get a chance. Once you approve, I'll get someone from the team to approve. Thanks

**Review by beccamc — APPROVED** [review]

**Comment by luisquintanilla** [comment]
@dotnet/docs this PR has been approved. Can I get an additional approval please. Thanks

File: docs/machine-learning/resources/azure-training-concepts-model-builder.md
Comment (gewarren) [inline docs/machine-learning/resources/azure-training-concepts-model-builder.md:50]: Not sure this makes sense. How about:
```suggestion
  - Dedicated: Suited for tasks of any duration, but especially long-running jobs. Not impacted by interruptions or lack of availability. Usually costs more because it reserves a dedicated set of compute resources in Azure for your tasks.
```

**Review by gewarren — APPROVED** [review]

File: docs/machine-learning/resources/azure-training-concepts-model-builder.md
Comment (luisquintanilla) [reply docs/machine-learning/resources/azure-training-concepts-model-builder.md:50]: Thanks @gewarren. Looks better :)

**Review by briacht — APPROVED** [review]
LGTM

**Pull request merged**
